test/test_tamper_evident.py

```diff
@@ -73,7 +73,7 @@ def test_editing_is_detected(self):
         self.assertFalse(tef.validate())
 
     def test_oneline_file_is_detected(self):
-        filename = self.write_tamper_evident(b"Am I OK?")
+        filename = self.write_tamper_evident("Am I OK?")
```

Comment:
Actually, wtf? I made changes in 6c728fb ("Make it work on Python 3") to change everything to explicit bytes.  The tests passed.  Then commit 0c34437 ("Make edx-lint commands work with both python2 and python3") changed it to explicit Unicode, and the tests fail.

@cpennington Why didn't we stick with explicit bytes?